### ISSUE TITLE
Problem with date validation ?

### ISSUE THREAD

**Issue by jdriesen** [ghiscoding/angular-validation#124]
Hi,
Thanks for this great code !
It seems there is a small issue with the EURO date-validation
I'm using an input field with: 
```javascript
...
validation	 = "date_euro_long_between:01/01/1920,01/01/2016|required"
...
```
Entering "31/02/1971" (February 31st 1971) seems to be accepted as a valid date, though Feb 31st doesn't exist... (and never existed, as far as I know...) : )

Regards,
Johnny Driesen

**Comment by ghiscoding** [comment]
Hello and thanks, unfortunately I'm using regular expression for all the dates and they are available as validators only for the formatting. I would really recommend you to use a date picker instead. 

Unless you know of a full regular expression that checks for that, which I unfortunately doubt. If you do find the expression then please make a PR. The regular expressions are all in the [validation-rules](https://github.com/ghiscoding/angular-validation/blob/master/src/validation-rules.js) file.

So basically I can't and won't fix this issue

**Comment by jdriesen** [comment]
Just for your information...
Found this one ...
[regular-expressions.info/dates.html](http://www.regular-expressions.info/dates.html)

Kind regards,
Johnny

**Comment by ghiscoding** [comment]
Hello, I did look at it but it seems that it needs an extra function (in that example VB) to check for the February problem. It doesn't look like it will be working :(
